src/controllers/organizationController.ts

```diff
@@ -150,6 +153,56 @@ export default class OrganizationController {
     });
   }
 
+  public getOrganizations(
+    req: Request
+  ): Promise<
+    // tslint:disable-next-line:max-union-size
+    | IResponseErrorValidation
+    | IResponseErrorInternal
+    | IResponseErrorForbiddenNotAuthorized
+    | IResponseSuccessJson<GetOrganizationResults>
+  > {
+    return withUserFromRequest(req, async user => {
+      const handleError = (error: Error) => {
+        log.error(
+          "An error occurred while reading from the database. %s",
+          error
+        );
+        return ResponseErrorInternal(
+          "An error occurred while reading from the database."
+        );
+      };
+      switch (user.role) {
+        case UserRoleEnum.ADMIN: // can see all the organizations
+          const errorOrOrganizations = await getAllRegisteredOrganizations();
+          return errorOrOrganizations.fold<
+            | IResponseErrorInternal
+            | IResponseSuccessJson<{ items: ReadonlyArray<Organization> }>
```

Comment:
this must be the type generated from the API spec (currently: `GetOrganizationResults`)